client/src/components/sidebar/nav/NavItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { NavLink } from 'react-router-dom';
+import { styled } from 'styled-components';
+
+import { Icon } from '../../Icon';
+
+interface NavItemProperties {
+  href: string;
+  label: string;
+}
+
+const NavItem: React.FC<NavItemProperties> = ({ label, href }) => {
+  return (
+    <ListItem>
+      <NavLink to={href}>
+        <Icon icon="board" /> <span>{label}</span>
+      </NavLink>
+    </ListItem>
+  );
+};
+export { NavItem };
+
+const ListItem = styled.li`
+  border-top-right-radius: 6.25rem;
+  border-bottom-right-radius: 6.25rem;
+  color: ${({ theme }): string => theme.palette.grey[500]};
+
+  a {
+    padding: 0.875rem 1.5rem;
+    display: grid;
+    grid-template-columns: 1rem auto;
+    align-items: center;
+    gap: 0.75rem;
+    font-weight: 700;
+  }
```

Comment:
i don't understand why in li tag we have styles for a tag. same as in ListButtonItem component

Reply:
because li tag styles depend on a tag state, I combined these styles in one wrapper. ListButtonItem has a Link component and NavItem has a NavLink instead. They both have an a tag under the hood, so I decided to write styles in a wrapper and refer to it

Reply:
Honestly, i don't like an idea to use it a tag both in this components. I understood your suggestion, but maybe is there a better way to handle it?

Reply:
I will try to handle it in another way and we'll see